reconcile/utils/slack_api.py

```diff
@@ -252,7 +259,7 @@ def update_usergroup_users(self, id: str,
                 raise
 
     def get_random_deleted_user(self):
-        for user_id, user_data in self._get('users').items():
+        for user_id, user_data in self.get_users():
```

Comment:
i don't see the `get_users` method... am i missing it?

Reply:
Let me add a test for this...